Apps/CommonData/src/ViewModels/UserProfileModel.cs

```diff
@@ -107,7 +107,7 @@ public UserProfileModel(IDictionary<string, UserPreferenceModel> preferences, IL
         /// <summary>
         /// Gets or sets a value indicating whether the user should be prompted of new tour slides.
         /// </summary>
-        public bool HasTourUpdated { get; set; }
+        public bool? HasTourUpdated { get; set; }
```

Comment:
Why does this need to be nullable?

Reply:
Until we actually set the value in the database it could be null. If it's null we default to false.

Reply:
It defaults to false, so this isn't necessary.

Reply:
its nullable to prevent issues on CreateUserProfile calls. The model is used in bidirectional binding so I opted for nullable